docs/osf-trivia-list/dts.md

```diff
@@ -103,7 +103,14 @@ you can help by running it in verbose mode and providing logs. To do that,
 follow the instructions below:
 
 1. Boot DTS.
-2. In the DTS menu enable verbose mode by typing `V` and pressing `Enter`.
-3. Enable sending logs by typing `L` and pressing `Enter`.
+2. In the DTS menu enable verbose mode by pressing `V`.
+3. Enable sending logs by pressing `L`.
 4. Reproduce the problem.
 5. The logs will be sent automatically after exiting from the menu.
+
+!!! tip
```

Comment:
@PLangowski fyi - added admonition block: https://python-markdown.github.io/extensions/admonition/
Recommened: rST suggests the following “types”: attention, caution, danger, error, hint, important, note, tip, and warning; however, you’re free to use whatever you want.